src/components/snackbar/snackbar.jsx

```diff
@@ -1,28 +1,29 @@
-import React, { Component } from "react";
+import React, { Component, useState } from "react";
 import {
   Snackbar,
   IconButton,
   Button,
   Typography,
-  SvgIcon
-} from '@material-ui/core';
+  SvgIcon,
+  LinearProgress,
```

Comment:
LinearProgress by MaterialUI

That blue bar on the snackbar when deposit/withdraw is executed